src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -70,31 +70,26 @@ public ReplicatorTest()
             ReopenDB();
             Database.Delete("otherdb", Directory);
             _otherDB = OpenDB("otherdb");
-            Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
+            //uncomment the code below when you need to see more detail log
+            //Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
         }
 
         [Fact]
-        public async Task TestCloseServerWhenReplicatorExistsAsync()
+        public async Task TestToCheckReplicatorStopsWhenAttemptToConnectToInvalidEndpoint()
         {
             var targetEndpoint = new URLEndpoint(new Uri("ws://192.168.0.11:4984/app"));
             var config = new ReplicatorConfiguration(Db, targetEndpoint);
             using (var repl = new Replicator(config))
             {
                 repl.Start();
                 var count = 0;
-
-                this.Invoking(x => ReopenDB())
-                    .ShouldThrow<CouchbaseLiteException>(
-                        "because the database cannot be closed while replication is running");
-            
                 while (count++ > 20 && repl.Status.Activity != ReplicatorActivityLevel.Stopped)
                 {
                     WriteLine($"Replication status still {repl.Status.Activity}, waiting for stopped...");
                     await Task.Delay(500);
                 }
 
-                if(repl.Status.Activity == ReplicatorActivityLevel.Stopped)
-                    WriteLine($"Replication status still {repl.Status.Activity}, stopped");
+                count.Should().BeLessThan(20);
```

Comment:
Right now the answer is clear as to why the count should be less than 20, but in the future we will probably forget, so it is best to add the optional reason into this assert:

`count.Should().BeLessThan(20, "because otherwise the replicator never stopped");`

This creates a single left to right readable line that explains what the assert is for.